src/routes/expenses.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const express = require('express');
+const expensesController = require('./controllers/users.js');
+
+const router = express.Router();
+
+router.get('/', expensesController.getAll);
+
+router.get('/:expensId', expensesController.getOne);
```

Comment:
Typo in parameter name 'expensId'. Consider changing it to 'expenseId' for better readability.